addons/website/models/website.py

```diff
@@ -138,6 +148,8 @@ def create(self, vals):
 
     @api.multi
     def write(self, values):
+        tools.update_field_value_to_ico(values)
```

Comment:
I just moved it to the website model just below the create/write so it's easy to see what it does without having to navigate multiple files.

It wasn't really necessary in tools, I don't think it will be used by anything else than website.